### ISSUE TITLE
[Question/Feature Request] Are window rules supported?

### ISSUE THREAD

**Issue by IceDBorn** [pop-os/cosmic-epoch#449]

**Comment by Department-stack** [comment]
I dont think so. if there is you would have to edit the config files directly. they will likely add something like it in the future but who knows exactly when

**Comment by leviport** [comment]
tracking in https://github.com/pop-os/cosmic-epoch/issues/847
